pkg/minikube/cruntime/crio.go

```diff
@@ -370,30 +378,3 @@ func crioImagesPreloaded(runner command.Runner, images []string) bool {
 func (r *CRIO) ImagesPreloaded(images []string) bool {
 	return crioImagesPreloaded(r.Runner, images)
 }
-
-// UpdateCRIONet updates CRIO CNI network configuration and restarts it
-func UpdateCRIONet(r CommandRunner, cidr string) error {
-	klog.Infof("Updating CRIO to use CIDR: %q", cidr)
-	ip, net, err := net.ParseCIDR(cidr)
-	if err != nil {
-		return errors.Wrap(err, "parse cidr")
-	}
-
-	oldNet := "10.88.0.0/16"
```

Comment:
Are we sure we dont need this ? 
@afbjorklund

Reply:
I think the idea was to stop kubernetes and podman/crio from trying to use the same network.

Some conflict with the default `10.88.0.0/16` subnet.

Reply:
@prezha shouldn’t we keep this ? To avoid the conflict ?